packages/manager/apps/dedicated/client/app/dedicatedCloud/users/activeDirectories/edit/federation-edit.routes.js

```diff
@@ -0,0 +1,20 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicatedCloud.details.users.federation-edit', {
+    url: '/federation/edit',
+    params: {
+      activeDirectory: null,
+    },
+    views: {
+      modal: {
+        component: 'dedicatedCloudUsersFederationEdit',
```

Comment:
to update regarding my previous comment.

Reply:
It's applied.